src/ring/middleware/oauth2.clj

```diff
@@ -106,9 +108,13 @@
                                   (assoc-in [::access-tokens id] access-token)
                                   (dissoc ::state)))))))))
 
+(defn- coerce-tokens-expires-date [tokens]
+  (into {} (for [[id t] tokens]
+             [id (update t :expires coerce/from-date)])))
```

Comment:
Might be better to use:

```clojure
(persistent!
 (reduce-kv #(assoc %1 %2 (update %3 :expires coerce/from-date))
            (transient {})
            tokens)
```

That way we're not creating an intermediate seq, but instead adding directly to a transient map.